New Checkout Drop-in v1.4.0-beta2 (#273)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/login-form.mdx

import OptionsTable from '@components/OptionsTable.astro';

> The `LoginForm` container is designed to handle user email input and validation within the checkout process.

Reviewer: ```suggestion
The `LoginForm` container handles user email input and validation within the checkout process.
```

}
```

> <li>The `displayTitle (*)` property is inherited from `TitleProps` interface; this is used to decide whether the title is displayed or not.</li>

Reviewer: ```suggestion
<li>The `displayTitle (*)` property inherits from the `TitleProps` interface to display or hide the title.</li>
```

> <li>The `displayHeadingContent` property set to _true_ displays the heading content with the sign-in/sign-out button.</li>

Reviewer: ```suggestion
<li>Set the `displayHeadingContent` property to _true_ to display the heading content with the sign-in/sign-out button.</li>
```

<li>The `slots` property is an object containing the following properties:</li>

#### Title property (*)

> The `Title` property is inherited as a result of merging the `slots` from `LoginFormProps` and `TitleProps`; this is a handler used to render a custom title content.

Reviewer: ```suggestion
Use the `Title` property to render a custom title. This property inherits from `LoginFormProps` and `TitleProps`.
```
